bgpd/bgp_clist.c

```diff
@@ -1117,11 +1117,12 @@ int lcommunity_list_set(struct community_list_handler *ch, const char *name,
 	}
 
 	if (str) {
-		if (!lcommunity_list_valid(str, style))
```

Comment:
`lcommunity_list_valid` is already called from `lcommunity_str2com`, no need to do this twice.
You should just delete this function call completely.

Reply:
Thank you for bringing this up. I will make the changes accordingly.